java/java.api.common/src/org/netbeans/modules/java/api/common/package-info.java

```diff
@@ -0,0 +1,13 @@
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+@TemplateRegistration(folder = "Other", iconBase = JavaTemplates.JAVA_ICON, displayName = "#JAVAtemplate_displayName", content = "resources/File.java.template", requireProject = false, targetName = "newJavaFile", description = "resources/Description.html", scriptEngine = "freemarker", category = {"java-classes", "java-classes-basic"})
+@Messages(value = "JAVAtemplate_displayName=Java File")
+package org.netbeans.modules.java.api.common;
```

Comment:
Does this need to be in an exported package? If not I would move this to the `org.netbeans.modules.java.api.common.impl` package.